modules/edge-aws-day2-extend-subnet.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: CONCEPT
+[id="extend_existing_to_local_zones_day2_subnet"]
+= Create a Local Zone subnet
```

Comment:
Hi @mtulio . We updated a section like this in the Day 0 docs, so can I reuse that module in place of your suggested module on this PR. See: https://63729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone#installation-creating-aws-subnet-localzone_installing-aws-localzone